tensorboard/backend/application_test.py

```diff
@@ -347,7 +347,7 @@ def _test(self, route, should_be_okay):
             plugin_name='foo',
             is_active_value=True,
             routes_mapping={route: lambda environ, start_response: None}),
-    ]
+      ]
```

Comment:
Please revert (inconsistent indentation, and unrelated change).

Reply:
Done